h/templates/app.html.jinja2

```diff
@@ -49,6 +50,10 @@
                 on-close="accountDialog.visible=false"></login-form>
       <sidebar-tutorial ng-if="isSidebar"></sidebar-tutorial>
       <share-dialog ng-show="shareDialog.visible"></share-dialog>
+      <about-this-version-dialog ng-show="aboutThisVersionDialog.visible"
```

Comment:
This should use `ng-if` rather than `ng-show`. The difference is that `ng-show` hides or shows the dialog using CSS depending on whether it is visible. `ng-if` doesn't put the element in the page at all if it is not visible. The benefit of removing the component from the page entirely is that Angular does not have to check for changes to the values of expressions in the `<about-this-version-dialog>` template whenever user input or a timeout or an HTTP request occurs (which might otherwise change the values of what is displayed in the dialog).